### DISCUSSION TITLE
Add tab in the bottom of the list

### DISCUSSION THREAD

**Discussion by smalibary** [zen-browser/desktop#1244] [Q&A]
hello,
for some reason i don't know why the add new tab in the top of the list in my desktop while it is in the bottom of the list in my laptop how can i move it to the bottom of the list??? 

![image](https://github.com/user-attachments/assets/f40074ff-a73b-449f-b1e9-b54f55211ef3)

**Comment by kwilcz** [comment]
Looks like one of the newer updates pushed that change. Would be great to have it as an option.

**Reply by smalibary** [reply]
Its weird i have been installing them in 1 day different ... in my laptop works perfectly but in pc seems there is a problem

**Reply by CcMenta** [reply]
This is not a bug, but there should be an option to change this.
